packages/esm-ward-app/src/types/index.ts

```diff
@@ -161,3 +163,24 @@ export interface EncounterRole extends OpenmrsResourceStrict {
   description?: string;
   retired?: boolean;
 }
+
+export interface PatientNotePayload {
+  encounterDatetime: string; // date and time the encounter was created (ISO8601 Long) (REQUIRED)
+  encounterType: string; // uuid of the encounter type - initial visit, return visit etc. (REQUIRED)
+  patient: string; // the patient to whom the encounter applies
+  location: string; // the location at which the encounter occurred (REQUIRED)
+  encounterProviders: Array<{ encounterRole: string; provider: string }>; // array of providers and their role within the encounter. At least 1 provider is required
+  obs: Array<ObsPayload>; // array of observations and values for the encounter
+  form?: string; // target form uuid to be filled for the encounter
+  orders?: Array<any>; // list of orders created during the encounter
+  visit?: string; // when creating an encounter for a specific visit, this specifies the visit
+}
+
+export interface ObsPayload {
```

Comment:
Is there an existing type defined in ESM Core we can use instead, otherwise fine to define it here. (fyi @ibacher @denniskigen )

Reply:
No @mogoodrich , I didn't find it,  but it should be present in the core.